hack/quickstart/init-master.sh

```diff
@@ -7,6 +7,7 @@ CLUSTER_DIR=${CLUSTER_DIR:-cluster}
 IDENT=${IDENT:-${HOME}/.ssh/id_rsa}
 SSH_OPTS=${SSH_OPTS:-}
 SELF_HOST_ETCD=${SELF_HOST_ETCD:-false}
+CLOUD_PROVIDER=${CLOUD_PROVIDER:-}
```

Comment:
It doesn't error, but i had assumed leaving it as the empty string would be the same as not having it at all.

The flag description however suggests that leaving it as the empty string will disable cloud-provider support rather then auto-detect by default. Now, that I've figured out the IAM permissions for AWS, I'll try testing without the flag and see if it auto-detects.

https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L240